src/cmd/destroy.go

```diff
@@ -44,7 +44,7 @@ var destroyCmd = &cobra.Command{
 		//       the scripts to remove k3s, we will still try to remove a locally installed k3s cluster
 		state, err := c.LoadZarfState(ctx)
 		if err != nil {
-			message.WarnErr(err, lang.ErrLoadState)
```

Comment:
Why removing `lang.ErrorLoadState`,

Reply:
Because I have moved this to a error message that is returned from the LoadZarfState function. That way we dont have to declare it everywhere.